mixer/pkg/attribute/list.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2018 Istio Authors
```

Comment:
So what is this type used for? Is this to implement eventual support for a STRING_LIST attribute type?

Reply:
Yes, the protobuf decoder relies on ability to put lists of repeated values. This is a hatch to keep the decoder aware of repeated vales, without fully implementing the list support (it's beyond the scope).